src/CosmosDB/CosmosDB/SQL/RemoveAzCosmosDBSqlContainer.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Text;
+using System.Linq;
+using Microsoft.Azure.Commands.CosmosDB.Models;
+using System.Reflection;
+using Microsoft.Rest.Azure;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.CosmosDB.Helpers;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+namespace Microsoft.Azure.Commands.CosmosDB
+{
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "CosmosDBSqlContainer", DefaultParameterSetName = NameParameterSet, SupportsShouldProcess = true), OutputType(typeof(void), typeof(bool))]
+    public class RemoveAzCosmosDBSqlContainer : AzureCosmosDBCmdletBase
+    {
+        [Parameter(Mandatory = false, ParameterSetName = NameParameterSet, HelpMessage = Constants.ResourceGroupNameHelpMessage)]
+        [ResourceGroupCompleter]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(Mandatory = false, ParameterSetName = NameParameterSet, HelpMessage = Constants.AccountNameHelpMessage)]
+        public string AccountName { get; set; }
+
+        [Parameter(Mandatory = false, ParameterSetName = NameParameterSet, HelpMessage = Constants.DatabaseNameHelpMessage)]
+        public string DatabaseName { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = Constants.ContainerNameHelpMessage)]
+        public string Name { get; set; }
+
+        [Parameter(Mandatory = false, ParameterSetName = ObjectParameterSet, HelpMessage = Constants.SqlContainerObjectHelpMessage)]
+        public PSSqlContainerGetResults InputObject { get; set; }
```

Comment:
InputObject should be mandatory for ObjectParameterSet.

Reply:
Changed as suggested.